polyfill/test/now/instant/length.js

```diff
@@ -16,7 +16,7 @@ includes: [propertyHelper.js]
 features: [Temporal]
```

Comment:
Should move this entire folder to `polyfill/test/Now/`, but do it separately, please.

Reply:
I'll assume that a separate commit in this PR is OK. LMK if not. Also was fun to to learn how to rename things case-sensitively in git, because of course it didn't work the easy way.